plugins/nodes/src/js/filters/NodesRegionFilter.ts

```diff
@@ -0,0 +1,21 @@
+import DSLFilterTypes from "#SRC/js/constants/DSLFilterTypes";
+import DSLFilter from "#SRC/js/structs/DSLFilter";
+
+/**
+ * This filter handles the `region:XXXX` for nodes
+ */
+const filter: DSLFilter = {
+  /*
+   * Handle all `region:XXXX` attribute filters that we can handle.
+   */
+  filterCanHandle: (filterType, filterArguments) =>
+    filterType === DSLFilterTypes.ATTRIB && filterArguments.label === "region",
```

Comment:
while reviewing this myself, i noticed that i may have broken the ability to apply multiple `is:`-filters when regions are involved. will investigate tomorrow.